src/list.tsx

```diff
@@ -53,27 +55,26 @@ interface CommonProps {
     disabled?: boolean;
     withChevron?: boolean;
     'aria-label'?: string;
+    right?: Right;
+    danger?: boolean;
 }
 
-type Right = (({centerY}: {centerY: boolean}) => React.ReactNode) | React.ReactNode;
-
 const renderRight = (right: Right, centerY: boolean) => {
     if (typeof right === 'function') return right?.({centerY});
 
     return centerY ? (
-        <div style={{display: 'flex', alignItems: 'center', height: '100%'}}>{right}</div>
+        <div style={{display: 'flex', alignItems: 'center', height: '100%'}}>
+            <div>{right}</div>
```

Comment:
wrapping this inside a div, because of [this case](https://mistica-web.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8AlCB3AOgOwAS4BcBLAgGxgF5MQAVE86nfCbABQCcYBnLi4ACgCUuCgD5cwAL6SmudkQDmACwJ9Z%2BJKPX5cSKEQBuogtwIBGJAHp9R7RpvHTAJisPtVrXlwzslzyAAaEAIlGABbbgQAbRAAWQgDIi4CAEN2EABdIPQiKBCuaIBmJwAGDMkgA). We don't want to force `display: flex` in the extra content by default. By wrapping `right` inside an interior div, we avoid this (but we center the content).